server/src/domain/game/Room.ts

```diff
@@ -26,6 +26,9 @@ export class RoomSettings {
 }
 
 export class Room {
+    gameMaster(gameMaster: any) {
```

Comment:
I don't think it's necessary, we already have `getGameMaster` getter

Reply:
yes, I don't even recall writing this so this must have been a linter auto-fix or something😅